forgettable-frontend/src/components/EncounterCardSummary/EncounterSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Card from '@mui/material/Card';
+import {Avatar} from '@mui/material';
+import './EncounterSummary.css';
+
+export default function EncounterSummary(props: { Name: any; Date: any; Description: any; Summary: any; }) {
+  const {
+    Name, Date, Description, Summary,
+  } = props;
+
+  return (
+    <div className="container">
+      <Card
+        className="card"
+        sx={{maxWidth: 220, maxHeight: 240, bgcolor: 'var(--lcard)', borderRadius: 6}}
+      >
+        <div className="header">
+          <Avatar
+            alt={Name}
+            src="Avatar.png"
```

Comment:
I think we can use the letter avatar in default but we can change this all together later!